castle/src/main/java/io/castle/android/Castle.java

```diff
@@ -12,28 +12,27 @@
 import android.os.Build;
 import android.os.Bundle;
 
-import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Locale;
 import java.util.Map;
 
+import io.castle.android.api.model.Custom;
 import io.castle.android.api.model.Event;
-import io.castle.android.api.model.IdentifyEvent;
 import io.castle.android.api.model.ScreenEvent;
+import io.castle.android.api.model.UserJwt;
 import io.castle.highwind.android.Highwind;
 import io.castle.android.queue.EventQueue;
 
 /**
  * This class is the main entry point for using the Castle SDK and provides methods for tracking events, screen views, manual flushing of the event queue, allowlisting behaviour and resetting.
  */
 public class Castle {
-    public static final String requestTokenHeaderName = "X-Castle-Request-Token";
     public static final String clientIdHeaderName = "X-Castle-Client-Id";
```

Comment:
What is this used for? This seems like an old header. We can drop backwards compatibility with this release given that we version it properly.

Reply:
X-Castle-Request-Token was used in previous versions but deprecated since a couple of releases back. That's why I removed it now.

Reply:
@sebastiansimson - why do we keep clientIdHeaderName ?